### DISCUSSION TITLE
Create Candelstick chart with node.js and save as png picture

### DISCUSSION THREAD

**Discussion by UsernameGitiHub** [tradingview/lightweight-charts#1463] [General]
I have little bit trouble to create candelstickc harts with node.js without using a browser, i just want to save the candelstick chart as picture and then i send the picture to a chat, but i have not found jet a solution, i have try chartjs but i dont get it working for candelstick, is it possible with your tradingview lightweight sdk to make that?

**Comment by spcfran** [comment]
Check out this recent discussion
https://github.com/tradingview/lightweight-charts/discussions/1415
